app/jsx/pages/PageBase.jsx

```diff
@@ -255,7 +255,8 @@ class PageBase extends React.Component
     if (this.state.adminLogin && this.state.adminLogin.loggedIn &&
         this.state.cmsModules && this.state.pageData &&
         this.state.permissions && this.state.permissions.admin &&
-        'header' in this.state.pageData && 'nav_bar' in this.state.pageData.header)
+        'header' in this.state.pageData && 'nav_bar' in this.state.pageData.header &&
+        !this.state.pageData.id)
```

Comment:
This line right here is really the only noteworthy change I made to this file. This turns off the drawer if it's an ItemPage